src/filters.py

```diff
@@ -0,0 +1,6 @@
+def use_n_most_occuring_users(df, n):
+    users = df.groupby(df['by']).size()
+    filtered_users, _ = zip(
+        *users.sort_values(ascending=False).head(100).items())
```

Comment:
Can you make the 100 a default parameter?

Reply:
This should not look this way. I fix it.

Reply:
What is the `n` doing?